installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -20,8 +20,27 @@ prefixPath: ""
 keptnSpecVersion: latest
 
 nats:
-  nameOverride: keptn-nats-cluster
-  nats.cluster.replicas: 3
+  nameOverride: keptn-nats
+  fullnameOverride: keptn-nats
+  cluster:
+    replicas: 3
+    name: nats
+  nats:
+    healthcheck:
+      startup:
+        enabled: false
```

Comment:
Needs to be disabled because `startupProbe` is not supported by Kubernetes version used in OpenShift